redisvl/index.py

```diff
@@ -57,32 +58,33 @@ def search(self, *args, **kwargs) -> List["Result"]:
         Returns:
             List[Result]: A list of search results
         """
-        return self._redis_conn.ft(self._name).search(*args, **kwargs)  # type: ignore
+        results: List["Result"] = self._redis_conn.ft(self._name).search(
+            *args, **kwargs
+        )
+        return results
```

Comment:
We should make a note about performance being better here since the results are not post processed.